sklearn/svm/classes.py

```diff
@@ -518,6 +518,14 @@ class SVC(BaseSVC):
         .. versionchanged:: 0.17
            Deprecated *decision_function_shape='ovo' and None*.
 
+    break_ties : bool, optional (default=False)
+        If true, `decision_function_shape`='ovr', and number of classes > 2,
+        `predict` will break ties according to the confidence values the same
```

Comment:
I'm not sure about "the same way"... `decision_function` doesn't break ties in that it doesn't choose a categorical best. Just say it uses `decision_function`, or that it is consistent with `decision_function`.

Reply:
the `decision_function` calls the `_ovr_decision_function` which says:
```
Compute a continuous, tie-breaking OvR decision function from OvO.

    It is important to include a continuous value, not only votes,
    to make computing AUC or calibration meaningful.
```

Are you saying it's actually not doing that?

Reply:
It is actually doing that, but it doesn't choose a category.

Reply:
I'd just say it will "break ties using the confidence values of `decision_function`"